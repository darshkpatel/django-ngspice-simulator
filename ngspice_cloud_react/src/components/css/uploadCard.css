.cardStyle {
    min-width: 70%;

}

.uploadCardBodyStyle{
    min-height: 50%;
    height: 50%;
    
}