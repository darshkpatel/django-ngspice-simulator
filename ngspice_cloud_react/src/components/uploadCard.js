import React from 'react';
import {
    Card,
    CardHeader,
    CardBody,
} from "shards-react";
import './css/uploadCard.css';
import { Container, Row } from "shards-react";
import UploadBox from "./uploadBox.js"

class UploadCard extends React.Component {
    render() {
        return (
            <Card className="cardStyle">
                <CardHeader>Upload Netlist for Circuit</CardHeader>
                <CardBody className="cardBodyStyle">
                    <Container>

                        <Row>
                        </Row>
                        <UploadBox />
                        <Row>

                        </Row>
                    </Container>
                </CardBody>
            </Card>

        );




    }
}

export default UploadCard;