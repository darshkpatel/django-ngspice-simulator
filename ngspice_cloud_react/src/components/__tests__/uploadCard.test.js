
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import uploadCard from '../uploadCard';


describe('uploadCard test', () => {
  it('uploadCard should match snapshot', () => {
    const component = renderer.create(<uploadCard
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
