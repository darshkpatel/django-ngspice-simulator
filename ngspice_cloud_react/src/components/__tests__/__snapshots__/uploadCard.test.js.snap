// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uploadCard test uploadCard should match snapshot 1`] = `<uploadCard />`;
